### ISSUE TITLE
MySqlError { ERROR 1210 (HY000): Incorrect arguments to mysqld_stmt_execute } occurs when an empty string is bound to a parameter

### ISSUE THREAD

**Issue by kjmrknsn** [blackbeam/rust-mysql-simple#129]
Hi, thanks for giving us such a nice crate.

I would like you to advise me something about the issue which I'm facing.

When I run the following code, I get `MySqlError { ERROR 1210 (HY000): Incorrect arguments to mysqld_stmt_execute }`.

```rust
pub fn update_title(transaction: &mut Transaction) -> Result<(), mysql::error::Error> {
    transaction.prep_exec(r"
        update notebook_history
        set
            title = :title
    ", params! {
        "title" => "",
    })?;

    Ok(())
}
```

I guess that if an empty string is bound to a parameter, such an error occurs.

Is there any way to bind an empty string to a parameter without having an error?

Thanks.

**Comment by blackbeam** [comment]
Thanks for report.
Fixed in v12.1.0

**Comment by kjmrknsn** [comment]
Thanks.
